src/client/graphics/layers/UILayer.ts

```
@@ -118,6 +185,63 @@ export class UILayer implements Layer {
+      this.lastMousePosition.x,
+      this.lastMousePosition.y,
+    );
+
+    this.game
+      .myPlayer()
+      ?.actions(tile)
+      ?.then((actions) => {
+        const unit = actions.buildableUnits.find(
+          (u) => u.type === this.lastSelectedBuildableUnit,
+        );
+        if (unit && unit.canBuild) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)